libc/include/llvm-libc-macros/stdbit-macros.h

```diff
@@ -55,6 +85,20 @@ inline unsigned stdc_leading_ones(unsigned long long x) {
       unsigned: stdc_leading_ones_ui,                                          \
       unsigned long: stdc_leading_ones_ul,                                     \
       unsigned long long: stdc_leading_ones_ull)(x)
+#define stdc_trailing_zeros(x)                                                 \
+  _Generic((x),                                                                \
```

Comment:
Hey! Thanks for doing this. Please make sure that you include `_BitInt(CHAR_BIT)`/`_BitInt(CHAR_BIT * 2)`/`_BitInt(CHAR_BIT * 4)`/`_BitInt(CHAR_BIT * 8)` versions of this, as that's part of the spec. (You can also just use `8/16/32/64`; I know that LLVM as a thing tends to not care about `CHAR_BIT != 8` platforms.)

Reply:
> Hey! Thanks for doing this.

Heh, thanks for standardizing these.  I was going to show you/ask for review at some point, I appreciate that you found this, took a look, and had feedback.  Do you have cycles at the moment to be cc'ed on more of these as I implement them? If not, no worries.

This is the first I've heard of `_BitInt`, is that something new-ish?

> 6.2.5 Types
> A bit-precise signed integer type is designated as _BitInt(N) where N is an integer constant expression
that specifies the number of bits that are used to represent the type, including the sign bit. Each
value of N designates a distinct type.42)

Yeah, woah gcc-14 and clang-14 both support this type...https://godbolt.org/z/or83fMcro 

Ok, I will add support for these. Will file an TODO and revisit; the initial implementations don't need to be perfect to land.

Reply:
Filed https://github.com/llvm/llvm-project/issues/80509 and will link to it from here.

Reply:
9a4eac509ec4